prow/jobs/kyma/kyma-metadata-governance.yaml

```diff
@@ -18,14 +18,16 @@ presubmits: # runs on PRs
           path_alias: github.com/kyma-project/test-infra
       spec:
         containers:
-          - image: eu.gcr.io/kyma-project/prow/test-infra/bootstrap:v20181121-f3ea5ce
+          - image: eu.gcr.io/kyma-project/prow/test-infra/bootstrap-gloang:v20181204-a6e79be
             securityContext:
```

Comment:
To prevent these kinds of problems, all the job descriptions have tests under `/test-infra/development/tools/jobs/`. Please add one for this job as well